worlds/sc2/Items.py

```diff
@@ -68,175 +241,727 @@ def get_full_item_list():
+    ItemNames.MARINE_OPTIMIZED_LOGISTICS:
+        ItemData(222 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 21, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MARINE, origin={"nco"},
+                 description="Increases Marine training speed."),
+    ItemNames.MEDIC_RESTORATION:
+        ItemData(223 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 22, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MEDIC, origin={"bw"},
+                 description="Removes negative status effects from target allied unit."),
+    ItemNames.MEDIC_OPTICAL_FLARE:
+        ItemData(224 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 23, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MEDIC, origin={"bw"},
+                 description="Reduces vision range of target enemy unit. Disables detection."),
+    ItemNames.MEDIC_OPTIMIZED_LOGISTICS:
+        ItemData(225 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 24, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MEDIC, origin={"bw"},
+                 description="Reduces Medic resource and supply cost."),
+    ItemNames.FIREBAT_PROGRESSIVE_STIMPACK:
+        ItemData(226 + SC2WOL_ITEM_ID_OFFSET, "Progressive Upgrade", 6, SC2Race.TERRAN,
+                 parent_item=ItemNames.FIREBAT, quantity=2, origin={"bw"},
+                 description=STIMPACK_LARGE_DESCRIPTION),
+    ItemNames.FIREBAT_OPTIMIZED_LOGISTICS:
+        ItemData(227 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 25, SC2Race.TERRAN,
+                 parent_item=ItemNames.FIREBAT, origin={"bw"},
+                 description="Reduces Firebat resource and supply cost."),
+    ItemNames.MARAUDER_PROGRESSIVE_STIMPACK:
+        ItemData(228 + SC2WOL_ITEM_ID_OFFSET, "Progressive Upgrade", 8, SC2Race.TERRAN,
+                 parent_item=ItemNames.MARAUDER, quantity=2, origin={"nco"},
+                 description=STIMPACK_LARGE_DESCRIPTION),
+    ItemNames.MARAUDER_LASER_TARGETING_SYSTEM:
+        ItemData(229 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 26, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MARAUDER, origin={"nco"},
+                 description=LASER_TARGETING_SYSTEMS_DESCRIPTION),
+    ItemNames.MARAUDER_MAGRAIL_MUNITIONS:
+        ItemData(230 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 27, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MARAUDER, origin={"nco"},
+                 description="Deals 20 damage to target unit. Autocast on attack with a cooldown."),
+    ItemNames.MARAUDER_INTERNAL_TECH_MODULE:
+        ItemData(231 + SC2WOL_ITEM_ID_OFFSET, "Armory 1", 28, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MARAUDER, origin={"nco"},
+                 description="Marauders can be trained without a Tech Lab attached to Barracks."),
```

Comment:
Internal tech module should probably get the `.format()` treatment. Something like:
`"{} can be trained from a {} without an attached Tech Lab"`

And on each description you feed the item name (plural) and the building. So here would be:
`INTERNAL_TECH_MODULE_DESCRIPTION.format("Marauders", "Barracks")`